Makefile

```diff
@@ -13,6 +13,10 @@
 # limitations under the License.
 
 BUNDLE_VERSION = 4.4.8
+# OC_VERSION and BUNDLE_VERSION are going to same for release artifacts but
+# different for nightly and CI bits where bundle version would be any random
+# string or dd-mm-yyyy format.
+OC_VERSION = 4.4.8
```

Comment:
This is better done as `OC_VERSION ?= $(BUNDLE_VERSION)` to avoid duplicating the bundle version a 3rd time. You can then run `make OC_VERSION=4.4.10` when you want to override it. However, it might be better to detect that we are in a CI/nightly context as opposed to a release context, and then use either `latest-4.x` or maybe https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.6.0-0.nightly-2020-06-30-020342/ to also test newest `oc` code and catch regressions early.

Reply:
> This is better done as OC_VERSION ?= $(BUNDLE_VERSION) to avoid duplicating the bundle version a 3rd time. You can then run make OC_VERSION=4.4.10 when you want to override it. 

Done

>  However, it might be better to detect that we are in a CI/nightly context as opposed to a release context, and then use either latest-4.x or maybe https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/4.6.0-0.nightly-2020-06-30-020342/ to also test newest oc code and catch regressions early.

yes we have to think that if this is possible because the base url is change here so to support that kind of thing we also need to make the base url configurable from the makefile.

Reply:
>  we also need to make the base url configurable from the makefile.

In my opinion, it's more useful to have a release/nightly toggle, or even autodetection in the Makefile.

Reply:
> In my opinion, it's more useful to have a release/nightly toggle, or even autodetection in the Makefile.

@cfergeau yes but somewhere we need to mention it, even we do release/nightly toggle then download url also need respect that choice, anyway we can take this as separate PR.

Reply:
Yes, I'm still of the opinion that even this OC_VERSION Makefile variable should have been part of this separate PR which would focus on adding CI/nightly support rather than slipping it in this not directly related PR.

Reply:
@cfergeau filled https://github.com/code-ready/crc/issues/1362 one to track it.